packages/app/src/app/pages/Sandbox/Editor/ForkFrozenSandboxModal/ForkFrozenSandboxModal.tsx

```diff
@@ -35,40 +28,50 @@ const ModalContent: React.FC = () => {
   };
 
   useKeyPressEvent('Enter', fork);
-
+  // <Close onClick={() => modalsActions.forkFrozenModal.close('cancel')} />
   return (
-    <Container>
-      <Close onClick={() => modalsActions.forkFrozenModal.close('cancel')} />
-      <Title>Frozen {customTemplate ? 'Template' : 'Sandbox'}</Title>
-      <p>
-        This {type} is frozen, which means you can’t make edits without
-        unfreezing it.
-      </p>
-      <p>Do you want to unfreeze the {type} for this session or make a fork?</p>
-      <Actions>
-        <UnlockButton onClick={unlock}>Unfreeze</UnlockButton>
-        <Button small onClick={fork}>
-          Fork
-          <Enter />
-        </Button>
-      </Actions>
-    </Container>
+    <Alert
+      title={`Frozen ${customTemplate ? 'Template' : 'Sandbox'}`}
+      description={
+        <>
+          <p>
+            This {type} is frozen, which means you can’t make edits without
+            unfreezing it.
+          </p>
+          <p>
+            Do you want to unfreeze the {type} for this session or make a fork?
+          </p>
+        </>
+      }
+      onCancel={unlock}
+      cancelMessage="Unfreeze"
+      onPrimaryAction={fork}
+      confirmMessage="Fork"
+    />
   );
 };
 
-export const ForkFrozenSandboxModal: React.FC = () => {
+const ForkFrozenSandboxModalComponent: React.FC<{ theme: any }> = ({
+  theme,
+}) => {
   const {
     state: { modals },
     actions: { modals: modalsActions },
```

Comment:
```suggestion
    actions: { modals: { forkFrozenModal: { close } } },
```

Reply:
I think its best to not do it that deep it may cause a lot of errors